### PR TITLE
Bio-Formats 6.3.1 release updates

### PR DESCRIPTION
**Pull request by dgault**
Add release announcement and bump for Bio-Formats 6.3.1
See https://trello.com/c/PdYccBlZ/20-open-pr-against-wwwopenmicroscopyorg

### CODE REVIEW

File: _posts/2019-12-16-bio-formats-6-3-1.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: Release of Bio-Formats 6.3.1
+intro-blurb: The OME team is pleased to announce the release of Bio-Formats 6.3.1
+---
+
+Today we are releasing Bio-Formats 6.3.1 which included the following changes:
```
Comment (mtbc) [inline _posts/2019-12-16-bio-formats-6-3-1.md]:
```suggestion
Today we are releasing Bio-Formats 6.3.1 which includes the following changes:
```

File: _posts/2019-12-16-bio-formats-6-3-1.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+* ICS (Image Cytometry Standard)
+  * prevented a potential error when writing ICS files with physical units which could not be converted
+* Imspector OBF
+  * fixed a bug with incorrect dimensions being parsed for some Imspector OBF files
+* Leica LAS AF LIF (Leica Image File Format)
+  * fixed a `NullPointerException` in some variants of the LIF file format
+* TIFF
+  * improved the performance of tiled writing
+* Zeiss CZI
+  * fixed issued with tile stitching and position size
```
Comment (mtbc) [inline _posts/2019-12-16-bio-formats-6-3-1.md]:
```suggestion
  * fixed issues with tile stitching and position size
```

File: _posts/2019-12-16-bio-formats-6-3-1.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+Bio-Formats tools improvements:
+
+* fixed a potential `NullPointerException` in SpringUtilities for Bio-Formats plugins (thanks to July Chen)
+* updated URL for fetching ImageJ upgrades in ``ijview``
+* fixed the XY coordinates for cropped images in ``bfconvert``
+* fixed a bug when using a cropped multi series file in ``bfconvert`` (thanks to Matthieu Moisse)
```
Comment (mtbc) [inline _posts/2019-12-16-bio-formats-6-3-1.md]:
```suggestion
* fixed a bug when using a cropped multi-series file in ``bfconvert`` (thanks to Matthieu Moisse)
```

**Comment by mtbc** [comment]
Can't see the card referenced in the description.

**Comment by mtbc** [comment]
Staged at https://snoopycrimecop.github.io/www.openmicroscopy.org/2019/12/16/bio-formats-6-3-1.html.

**Comment by mtbc** [comment]
Huh, I must be missing the obvious, I can't find where that contributors page is staged!

**Comment by dgault** [comment]
Contributors is at https://snoopycrimecop.github.io/www.openmicroscopy.org/contributors/

**Comment by mtbc** [comment]
Gah! Sorry, I was blind, I looked at that page but the next section started right below where the pane ended and I didn't realize there was more. :smiley:

File: contributors/index.html [hunk 4]
```diff
@@ -288,6 +292,10 @@ <h5 class="subheader">The Bio-Formats project has been fortunate to receive assi
             Claire McQuin <a target="_blank" href="https://www.broadinstitute.org">Broad Institute</a>
             </div>
             
+            <div class="large-expand columns">
+            Matthieu Moisse <a target="_blank" href="https://www.kuleuven.be">VIB KU LEUVEN</a>
```
Comment (mtbc) [inline contributors/index.html]:
```suggestion
            Matthieu Moisse <a target="_blank" href="https://www.kuleuven.be">VIB KU Leuven</a>
```

**Pull request merged**